### PR TITLE
stroke style directive

### PR DESCRIPTION
**Pull request by alienlike**

### CODE REVIEW

**Comment by alienlike** [comment]
Rebased and ready for review.

File: src/components/style.components.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import { Component, Directive, EventEmitter, Host, OnDestroy, OnChanges, AfterContentInit, Input, Output, ContentChild } from '@angular/core';
 import { style } from 'openlayers';
+import { Subscription } from 'rxjs/Subscription';
+import { Observable } from 'rxjs/Observable';
```
Comment (FallenRiteMonk) [inline src/components/style.components.ts]: Unused import Observable

File: src/components/style.components.ts [hunk 2]
```diff
@@ -52,12 +73,28 @@ export class StyleComponent implements OnDestroy, AfterContentInit {
 
     ngAfterContentInit() {
         this.update();
-        this.childSubscription$ = this.iconStyleDirective.onChanged.subscribe((): void => {
-            this.update();
-        });
+        if (this.iconStyleDirective) {
+            this.childSubscription$ = this.iconStyleDirective.onChanged.subscribe((): void => this.update());
+        }
+        else if (this.strokeStyleDirective) {
```
Comment (FallenRiteMonk) [inline src/components/style.components.ts]: Can stroke and icon not be used on the same style element?
If they can, this shouldn't be ```else if``` but only ```if```
same for the update function

**Comment by FallenRiteMonk** [comment]
The code looks good to me, but wasn't able to test it due to the lack of an example.
Please add a usage of stroke to the example, might also be good for other users.

**Comment by alienlike** [comment]
Fixed #30 if we agree on the relative path solution. Addresses comments above.

**Comment by FallenRiteMonk** [comment]
LGTM, I would also say that you can keep the relative path as a temporary solution since its more important that everybody can test the example than it being aot compatible, especially since we know that library is aot compatible at the moment. Then when we finished discussing the real solution in #30 we can change it again.

**Comment by quentin-ol** [comment]
should be addressed in 0.5.0, though in exactly the same fashion. Feel free to reopen if need be.

**Pull request closed without merging**